tests/infra_test.go

```diff
@@ -155,7 +157,9 @@ var _ = Describe("Infrastructure", func() {
 		tests.BeforeAll(func() {
 			tests.BeforeTestCleanup()
 
+			// to avoid testing a corner case, we use 2+ VMIs
```

Comment:
As a reviewer, I lack details about what corner case you are talking about, and I can't find it described in commit messages either. Please add some details because while code wise the change looks legit, it's not immediately clear why we introduce it. Thanks.

Reply:
Totally fair, I was just too cryptic. Fixed.